keyboards/handwired/zergo/readme.md

```diff
@@ -0,0 +1,7 @@
+# zErgo
+
+![zergo](https://lh3.googleusercontent.com/ebufdZluIsuej_2MMqICxgyStO1TVOQXD7g-Nb4iJhjHOiG_g3gAAH335jnDCo7crZJPiUMTLWUrQ8qJWk9wbCAbMjEaNanWuqNdkMfQ7h0Oq2AYgcDvVulS68AOG65g2vtjp1_wBaJjy2D0SqGoEibhxi5A9ehQxHUJmCUaTuNiHXGR02shFr8JF6wgP7K72OY4g58RKyOUnCDRoiYENuBcZzcT_dR2Zbmzyj9fqo7VBuMJv3ry4S1aM1JiUw2ctMabhAYbjFcp7DqhgGbl0bTbcQSpYiywDLyeJol7zU0tCgqT8KixXn1YbwphlgYBhw379KiMoBHuFzoQNjwsWRPK0lzgsNRE9j6KrCCwVEokSdrt9eTLHO5emM--J_F32JuF-eT__pQBBki0kU0nS0TZl-4TWY5L8L7cb8Awg-OEX4mxN_ONWAkxu3eSKuKPOWYUVwf-VVDhjHjfv74x8Eq84T4jqyXsMPmBZjmi_Xz5c2KUusC0fQYAJa5bP3FLjEd83IBA4n-B4fx1nVUtKZAZCkVXWgJNRrYRjZlY5kKHPfu9w-2T6Q_vLwNfr5FudEHlxfTAjecxGoFtgnRmCas1ZSf0UZ15FIAfO22mQlmiRdGZge_OXn4UuWOCLlRviBiZJTIbToKaYpgqlkqlg0j7nqx951xSCEMBjEL_GDi1IFZdm6RvQAPp=w830-h358-no?authuser=0)
+
+A compact 70% (15x6) keyboard inspired by the Katana60 project, but with F keys and adopted for the Russian layout.
+
+* Keyboard Maintainer: [Greenjack](https://github.com/greenjack-z)
```

Comment:
```suggestion
* Keyboard Maintainer: [Greenjack](https://github.com/greenjack-z)
* Hardware Supported: *The PCBs, controllers supported*
* Hardware Availability: *Links to where you can find this hardware*

Make example for this keyboard (after setting up your build environment):

    make handwired/zergo:default

Flashing example for this keyboard:

    make handwired/zergo:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Please fill out the Hardware supported and Hardware availability items.

Reply:
It's a prototype keyboard made by me using teensy 2.0++ controller and self made PCB - what exactly I have to list?